### PR TITLE
Clarify the /runs endpoint

### PR DESCRIPTION
**Pull request by alenkacz**
JIRA issues: METRONOME-182

### CODE REVIEW

**Comment by alenkacz** [comment]
low hanging 🍎

File: api/src/main/resources/public/api/v1/jobs.raml [hunk 1]
```diff
@@ -276,7 +276,7 @@ post:
     #
     get:
       description:
-        Get the list of all runs for this jobId.
+        Get the list currently active runs for this jobId. Once the run is finished, it will not be available through this endpoint.
```
Comment (meln1k) [inline api/src/main/resources/public/api/v1/jobs.raml]: `Get the list currently active` =>  `Get the list of currently active`

Comment (meichstedt) [reply api/src/main/resources/public/api/v1/jobs.raml]: agreed

Comment (alenkacz) [reply api/src/main/resources/public/api/v1/jobs.raml]: thanks!

**Pull request merged**